### PR TITLE
Module 1:  microverse_code_review_project

### PR DESCRIPTION
**Pull request by preciousakams**
### **Commit details**
All files copied from the survey project which I completed during the Admissions Trials

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+
+
+
+  <head>
+    <meta charset="UTF-8">
```
Comment (preciousakams) [inline index.html:7]: Kindly make sure to include viewport meta as it is part of the main structure of the HTML document.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  </p>
+
+<form id="survey-form">
+  <label for="name" id="name-label">Name</label>
+    <input id="name" placeholder="name" required>
+    <label for="email" id="email-label">Email</label>
+    <input type="email" id="email" placeholder="email" required>
+    <label for="number" id="number-label">Number</label>
+    <input type="number" id="number" min="7" max="15" required placeholder="number">
+    <select id="dropdown" name="dropdown">
```
Comment (preciousakams) [inline index.html:27]: Kindly add a title attribute to your select element by adding a label to it which will clearly define what it is used for.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    </select>
+    <br><br>
+    <input type="radio" name="group1" value="option 1">
+    <input type="radio" name="group1" value="option 1">
+    <input type="radio" name="group1" value="option 1">
+    <input type="radio" name="group1" value="option 1">
+    <br><br>
+    <input type="checkbox"value="check1">
+    <input type="checkbox"value="check2">
+    <textarea></textarea>
```
Comment (preciousakams) [inline index.html:40]: Kindly add a label or placeholder to your form elements to make it clear for anyone who sees your code.

**Review by preciousakams — COMMENTED** [review]
**Hello @kamzzy** 
You did a great job. 👏
there are some issues you need to work on in order to get to the next phase, but you are almost there.

 ### **Highlights** 👏👏
- You completed your project
- passed all requested tests
### **Request changes** ♻

-  Kindly add linters to your project, and ensure your code passes all tests.
- Kindly check my comments below and make all required changes. 

### **Optional changes** 👏👏
- Add some CSS styling to your page.

Thanks😊

File: index.html (hunk 1 above)
Comment (sja-thedude) [reply index.html:7]: - [ ] Kindly rephrase this with the reason why you asked for this. Like "Kindly make sure to include viewport meta as it is part of the main structure of the HTML document."

File: index.html (hunk 2 above)
Comment (sja-thedude) [reply index.html:27]: Kindly rephrase this with the reason why you asked for this. Like "Kindly add a title attribute to your select element by adding a label to it which will clearly define what it is used for."

File: index.html (hunk 3 above)
Comment (sja-thedude) [reply index.html:40]: - [ ] Kindly rephrase this with the reason why you asked for this. Like "Kindly add a label or placeholder to your form elements to make it clear for anyone who sees your code."

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @kamzzy, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good PR Title and Description

## Required 
- [ ] Kindly mention that linters are not set up for the project, it is required for linters to pass all the tests. 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @kamzzy, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Required Changes ♻️
- [ ] Kindly implement the changes asked previously for approval of the project. Make sure to edit your inline comments to the one asked.
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by preciousakams** [comment]
@sja-thedude, it's obvious you didn't read through my comments, because if you did you would notice I had rephrased all my comments as you requested, kindly read through and let me know which comment I missed out on.

**Review by Mre55 — APPROVED** [review]
**Hi @kamzzy,**

# Status: APPROVED  🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

-----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**